CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ endforeach(GRIDFILE)
 set(CPACK_SOURCE_GENERATOR "TGZ;ZIP")
 set(CPACK_INCLUDE_TOPLEVEL_DIRECTORY 0)
 set(CPACK_SOURCE_PACKAGE_FILE_NAME "proj-datumgrid-${PROJ_DATUMGRID_VERSION_MAJOR}.${PROJ_DATUMGRID_VERSION_MINOR}")
-set(CPACK_SOURCE_IGNORE_FILES "/europe/;/oceania/;/north-america/;/build/;/cmake/;CMakeLists.txt;/lla/;HOWTORELEASE;/.github/;/.git/;.swp$;.*~;${CPACK_SOURCE_IGNORE_FILES}")
+set(CPACK_SOURCE_IGNORE_FILES "/europe/;/oceania/;/north-america/;/build/;/cmake/;CMakeLists.txt;/lla/;HOWTORELEASE;/.github/;/.git/;.swp$;.*~;.*.py;.*.sh;${CPACK_SOURCE_IGNORE_FILES}")
```

Comment:
The `build_egm96_15_gtx` directory should also be excluded from the proj-datumgrid archives.

Reply:
Fixed in d6dd4023fc6592e241c92749bcbee466972c1352